tabs-tabs-tabs/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "manifest_version": 2,
+  "name": "Tabs, tabs, tabs",
+  "version": "0.1",
+  "applications": {
+    "gecko": {
+      "id": "tabs-tabs-tabs@mozilla.org"
+    }
+  },
+  "browser_action": {
+      "default_title": "Tabs, tabs, tabs",
+      "default_popup": "tabs.html"
+  },
+  "permissions": [
+      "tabs"
+  ]
+}
```

Comment:
In the other examples, we include manifest keys that are optional but are still good practice to include:
   * description
   * icons
   * homepage_url
   * strict_min_version in the applications key
   * default_icon in the browser_action key